deb/openmediavault/usr/share/php/openmediavault/email.inc

```diff
@@ -64,7 +64,7 @@ class Email {
 		if (!empty($this->from))
 			$headers[] = sprintf('From: %s', $this->from);
 		return mail($this->to, $this->subject, $this->message,
-		  implode("\r\n", $headers));
+			implode(PHP_EOL, $headers));
```

Comment:
This change did not change the behaviour of the mails being rejected by the mail provider. Anyway, according to the information you provided, it is better to keep PHP_EOL at this point. I could not observe any change in the behaviour of how the mails get formatted or placed in the queue.